src/components/shared/IbanFields.vue

```diff
@@ -375,17 +375,17 @@ watch( () => store.state.bankdata.values.iban, ( newIban: string ) => {
 			}
 		}
 	}
-}
 
-h2.icon-title {
-	padding: map.get(units.$spacing, 'small') 0 0 1.5rem;
-	margin: 0 map.get(units.$spacing, 'small');
-	font-size: 18px;
-	line-height: 25px;
+	h3.icon-title {
```

Comment:
Can you move this up and nested into the `.iban-calculator` and drop the h3 part of the selector? It means that if we ever have other `<h3 class="icon-title">` elements they won't be affected by this style

Reply:
I nested it inside of the .iban-calculator yesterday, I don't really know what other kinds of nesting you mean

Reply:
Oh it doesn't look nested in the Github diff, you still should remove the `h3`